rust/src/ffi/steppable_evmc_vm.rs

```diff
@@ -15,9 +15,9 @@ use crate::{
 };
 
 #[no_mangle]
-extern "C" fn evmc_create_steppable_evmrs() -> *const evmc_vm_steppable {
+extern "C" fn evmc_create_steppable_evmrs() -> *mut evmc_vm_steppable {
     let new_instance = evmc_vm_steppable {
-        vm: evmc_vm::evmc_create_evmrs() as *mut evmc_vm_t,
+        vm: evmc_vm::evmc_create_evmrs(),
```

Comment:
This is definitely one code smell less. Good to see when refactors make code more reasonable.